espnet2/speechlm/inference_utils.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Jinchuan Tian
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+
+import json
+import logging
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Union
+
+import torch
+import torchaudio
+from kaldiio import WriteHelper
+
+from espnet2.speechlm.definitions import SPEECHLM_TASKS
```

Comment:
Why is `SPEECHLM_TASKS` all upper-cases?
Any intention?
This breaks the naming convention.

Reply:
I want to emphasize that the SPEECHLM_TASKS are pre-defined hard codes shared in all SpeechLM modules, like MACROs. 
Would it be ok to keep it?